.github/workflows/ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
         run: |
           pip install --upgrade pip
           sudo apt remove ansible
-          pip install ansible-core
+          pip install git+https://github.com/ansible/ansible.git
```

Comment:
Do these changes not work anymore with the released version?

Reply:
it works, I changed because I believe @Spredzy uses the github version on his CI, so I'm trying to keep the same version

Reply:
Yes, we do use @devel in our CI. Thanks for the fix.

I think, ideally, CI should be testing with both:

  * `pip install ansible-core`: To ensure it works with the version currently available
  * `pip install git+https://github.com/ansible/ansible.git`: To ensure any incompatible change can be spotted in advance (and not *after* a release of ansible-core)
  
  I don't know the level of changes that would require in that file, but that would provide a good amount of coverage.